packages/providers/src/provider.ts

```diff
@@ -64,30 +64,32 @@ export type ContractResult = {
   bytecode: string;
 };
 
+type ConsensusParameters = {
```

Comment:
is it possible/good to use the generated type from graphql?

Reply:
@LuizAsFight the type coming directly from graphql has all the fields with the type `string`. This is why I am using the types returned from `procesGqlChain`'s results - `ChainInfo`: https://github.com/FuelLabs/fuels-ts/blob/ff8c4dfeafd23744f2a6f747910975540cd381cd/packages/providers/src/provider.ts#L70